docs/development.md

```diff
@@ -31,23 +31,57 @@ make test-e2e-go-ephemeral-multi-repo
 ```
 
 ## Build
-The project can be built from source with a single command:
+Config Sync can be built from source with a single command:
 
 ```
-make build-oss
+make config-sync-manifest
 ```
 
 This will build all the docker images needed for Config Sync and generate
 the manifests needed to run it. The images will by default be uploaded to 
 Google Container Registry under your current gcloud project and the manifests
-will be created in .output/oss/manifests under the Config Sync directory.
+will be created in .output/staging/oss under the Config Sync directory.
```

Comment:
In #189 I explored using Kustomize to bundle all of the OSS manifests into a single one, to make it easier to e.g. copy out into some other location and make sure you got everything. Is it worth including that approach here?

Also, when writing that PR I noticed that [one of the required manifests](https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/manifests/ns-reconciler-cluster-role.yaml) was not included. I don't see anything addressing that in this PR!

Reply:
This target is already using the [append_manifests](https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/Makefile.build#L257) to consolidate into a single manifest. I think we could consider replacing append_manifests with kustomize as you proposed but that should be a separate change.

This target also already includes `ns-reconciler-cluster-role`